File: Solution/CSharp/119.Pascal'sTriangleII.cs
public class Solution {
    public IList<int> GetRow(int rowIndex) {
        List<int> result = new List<int>();
        if (rowIndex >= 0) { result.Add(1); }
        if (rowIndex >= 1) { result.Add(1); }
        for (int index = 2; index <= rowIndex; index++) {
            result.Add(1);
            for (int i = index - 1; i >= 1; i--) {
                result[i] = result[i] + result[i - 1];
            }
        }
        return result;
    }
}

File: Solution/Python/119. Pascal's Triangle II.py
class Solution:
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        res = []
        if rowIndex >= 0:
            res.append(1)
        if rowIndex >= 1:
            res.append(1)
        for idx in range(2, rowIndex + 1):
            res.append(1)
            for i in range(idx - 1, 0, -1):
                res[i] = res[i] + res[i - 1]
        return res


rowIndex = 4
res = Solution().getRow(rowIndex)
print(res)
